R/stan-likelihood.R

```diff
@@ -1401,3 +1401,11 @@ tp <- function(wsp = 2) {
   wsp <- collapse(rep(" ", wsp))
   paste0(wsp, "target += ")
 }
+
+## FIXME: IK, 30 Oct 2024: what should the default values of reqn and
+## vec be in order to be benefitting from vectorization when possible?
+stan_log_lik_xbetax <- function(bterms, ...) {
+    p <- stan_log_lik_dpars(bterms, reqn = TRUE)
+    sdist("xbetax", p$mu, p$phi, p$kappa, vec = FALSE)
```

Comment:
if you have all vectorized versions really implemented and want to test them, you can set reqn = FALSE and vec = TRUE. At least for your testing this will work.

Reply:
See https://github.com/paul-buerkner/brms/pull/1698#discussion_r1824060648_